app/controllers/admin/badges_controller.rb

```diff
@@ -131,7 +131,8 @@ def update
   def destroy
     badge = find_badge
     StaffActionLogger.new(current_user).log_badge_deletion(badge)
```

Comment:
I feel like these 3 calls should now be in a `Badge.transaction do` block now, it would be awkward if we cleared all the titles but then failed to destroy the badge. Should be all-or-nothing.